pkg/storage/bloom/v1/bloom_tokenizer.go

```diff
@@ -115,6 +115,7 @@ func newMetrics(r prometheus.Registerer, namespace, subsystem string) *metrics {
 }
 
 func clearCache(cache map[string]interface{}) {
+	// TODO: Once we move Loki to Go 1.21, we can just use clear() here.
```

Comment:
We already require 1.21 in the go.mod

Reply:
argh, it was 1.20 when I made the PR.. thanks